packages/moti/src/core/types.ts

```diff
@@ -24,21 +7,13 @@ import type {
   DerivedValue,
 } from 'react-native-reanimated'
 
-export type Transforms = PerspectiveTransform &
-  RotateTransform &
-  RotateXTransform &
-  RotateYTransform &
-  RotateZTransform &
-  ScaleTransform &
-  ScaleXTransform &
-  ScaleYTransform &
-  TranslateXTransform &
-  TranslateYTransform &
-  SkewXTransform &
-  SkewYTransform
+type Mutable<T> = T extends object ? { -readonly [P in keyof T]: T[P] } : never
+
+export type Transforms = Mutable<
+  Exclude<ViewStyle['transform'], string | void>[number]
+>
```

Comment:
The transform types don't seem to exist in RN 0.81

I derive them from the ViewStyle type now.